charts/gateway/templates/secret.yaml

```diff
@@ -21,13 +21,15 @@ metadata:
 {{- end }}
 type: Opaque
 data:
+{{- if .Values.disklessConfig }}
```

Comment:
I suggest we change this secret to include a node.properties if diskless config is false. This would reuse the default credentials in values.yaml.

This would keep the demo/trying out the chart experience the same.

Reply:
@Gazza7205 , we are creating secret from node.properties via [node-properties-secret.yaml](https://github.com/CAAPIM/apim-charts/blob/b2e9a530e4df19f73d1211003157493638647215/charts/gateway/templates/node-properties-secret.yaml ) , when disklessConfig is disabled.